Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -78,75 +108,514 @@ public static void ShowWindow()
+                + $"\n\nXR SDK/XR Management - Choose this if you want to have a smoother upgrade path to Unity 2020. Supports HoloLens, Windows Mixed Reality/Oculus/Magic Leap headset. Note: do NOT choose this if you anticipate using Azure Spatial Anchors as there is a known compatibility issue."
+#else
+                +$"Unity currently provides the Legacy XR pipeline in this version ({Application.unityVersion}). Please click on the Enable Legacy XR button if you are targeting AR/VR headsets (e.g. HoloLens, Windows Mixed Reality headset, OpenVR headset etc.). "
+#endif // UNITY_2019_3_OR_NEWER
+                + "\n\n\nYou may also skip this step and configure manually later.", EditorStyles.wordWrappedLabel);
+            CreateSpace(15);
+            using (new EditorGUILayout.HorizontalScope())
+            {
+#if UNITY_2019_3_OR_NEWER
+                if (GUILayout.Button("Legacy XR (Recommended)"))
+#else
+                if (GUILayout.Button("Enable Legacy XR"))
+#endif // UNITY_2019_3_OR_NEWER
+
+                {
+                    XRSettingsUtilities.LegacyXREnabled = true;
+                }
+#if UNITY_2019_3_OR_NEWER
+                if (GUILayout.Button("XR SDK/XR Management"))
+                {
+                    if (!XRSettingsUtilities.XRManagementPresent)
+                    {
+                        AddUPMPackage("com.unity.xr.management");
+                    }
+                    currentStage = ConfigurationStage.InstallBuiltinPlugin;
+                    Repaint();
+                }
+#endif // UNITY_2019_3_OR_NEWER
+                if (GUILayout.Button("Learn more"))
+                {
+                    Application.OpenURL(XRPipelineDocsUrl);
+                }
+            }
+            RenderSetupLaterSection(true, () => {
+                currentStage = ConfigurationStage.ProjectConfiguration;
+                Repaint();
+            });
         }
 
-        private void CompilationPipeline_assemblyCompilationStarted(string obj)
+        private void RenderLegacyXRPipelineDetected()
         {
-            // There should be only one pop-up window which is generally tracked by IsOpen
-            // However, when recompiling, Unity will call OnDestroy for this window but not actually destroy the editor window
-            // This ensure we have a clean close on recompiles when this EditorWindow was open beforehand
-            Close();
+            EditorGUILayout.LabelField("XR Pipeline Setting - LegacyXR in use", EditorStyles.boldLabel);
+            EditorGUILayout.LabelField("To build applications for AR/VR headsets you need to specify an XR pipeline. "
+                + $"\n\nThe LegacyXR pipeline is detected in the project. Please be aware that the LegacyXR pipeline is deprecated in Unity 2019 and is removed in Unity 2020."
```

Comment:
```suggestion
                + $"\n\nThe LegacyXR pipeline is detected in the project. Please be aware that the LegacyXR pipeline is deprecated in Unity 2019 and will be removed in Unity 2020."
```

Reply:
I am not sure if "will be" would be better here as the Legacy XR pipeline has already been removed in Unity 2020.